presto-main/src/main/java/io/prestosql/sql/planner/SubqueryPlanner.java

```diff
@@ -261,22 +261,13 @@ private PlanBuilder appendExistSubqueryApplyNode(PlanBuilder subPlan, ExistsPred
 
         PlanBuilder subqueryPlan = createPlanBuilder(existsPredicate.getSubquery());
 
-        PlanNode subqueryPlanRoot = subqueryPlan.getRoot();
-        if (isAggregationWithEmptyGroupBy(subqueryPlanRoot)) {
```

Comment:
Are there any more usages of `isAggregationWithEmptyGroupBy`?

Reply:
No, that's why I removed the method, too.